src/components/plots/barPlot.ts

```diff
@@ -46,6 +46,32 @@ export module Plot {
     }
 
     /**
+     * Returns true if the supplied coordinates or Extents intersect or are contained by bbox.
+     */
+    private static _intersectsBBox(xValOrExtent: number | Extent, yValOrExtent: number | Extent,
+      bbox: SVGRect, tolerance = 0.5) {
+      var xExtent: Extent = Bar._parseExtent(xValOrExtent);
+      var yExtent: Extent = Bar._parseExtent(yValOrExtent);
+
+      // SVGRects are positioned with sub-pixel accuracy (the default unit
+      // for the x, y, height & width attributes), but user selections (e.g. via
+      // mouse events) usually have pixel accuracy. A tolerance of half-a-pixel
+      // seems appropriate.
```

Comment:
I think the comment might be helpful here, if only because we're overloading the term "intersects"? In general, is being inside of a SVGRect typically considered intersection? If you think it's clear without the comment, I'll remove it.